src/dealDashboard/dealMenubar/dealMenubar.ts

```diff
@@ -19,12 +19,12 @@ export class DealMenubar {
     this.eventAggregator.publish("handleInfo", "Deal URL copied");
   }
 
-  @computedFrom("deal.status", "deal.isCancelled")
+  @computedFrom("deal.status", "deal.isCancelled", "deal.isUserProposalLead")
```

Comment:
should be: "deal.isUserProposalLead", "deal.isCancelled", "deal.isExecuted" (remove status, add isExecuted)